index.js

```diff
@@ -0,0 +1,122 @@
+const teachers = {
+  heading: 'Our Teachers',
+  title: 'SehYeong Noh',
+  description: 'Director of Frontend department and member of University board.',
+  descriptionn: 'SehYeong Noh has attended his Masters and Phd degree in this university and He was amongest the best achiveres and he is currently Teaching in this university.',
+  img: './img/teach.png',
+  alt: 'teacher',
+};
```

Comment:
- [ ] Kindly add the other 5 speakers as you see in the given design to match and enhance the appearance of your website. You can choose different speakers as you wish.
yours:
![image](https://user-images.githubusercontent.com/91486791/235131829-4f80bf7d-76da-4340-915d-1748b3ea079a.png)
given:
![image](https://user-images.githubusercontent.com/91486791/235131900-b4b1f16d-baa5-42a9-afc6-a0bed404a898.png)